app/controllers/subject_viewer.coffee

```diff
@@ -53,6 +54,17 @@ class SubjectViewer extends Controller
     @el.attr tabindex: 0
     @setClassification @classification
 
+  setClassificationCohort: (cohort) =>
+    @classification.cohort = cohort
```

Comment:
fair enough, i didn't realize we had a JSON interface too, good idea to persist it to that too.